test/bin/ci_phase_iso_boot.sh

```diff
@@ -27,8 +27,11 @@ bash -x ./bin/start_webserver.sh
 # Set up the storage pool for VMs
 bash -x ./bin/manage_vm_storage_pool.sh create
 
+# Set up the scenario sources
+[[ -z "${PERIODICS}" ]] && SCENARIO_SOURCES="scenarios/*.sh" || SCENARIO_SOURCES="scenarios/periodics/*.sh"
```

Comment:
Should this go to `common.sh` as it's already defined in 2 places?

Reply:
Done.